tests/integration/test_agentd/test_agentd_parametrized_reconnections.py

```diff
@@ -47,23 +49,26 @@
 
 def teardown():
     global remoted_server
-    if remoted_server != None:
+    if remoted_server is not None:
         remoted_server.stop()
 
 
 def set_debug_mode():
+    """
+    Set debug2 for agentd in local internal options file.
+    """
     if platform.system() == 'win32' or platform.system() == 'Windows':
         local_int_conf_path = os.path.join(WAZUH_PATH, 'local_internal_options.conf')
         debug_line = 'windows.debug=2\n'
     else:
         local_int_conf_path = os.path.join(WAZUH_PATH, 'etc', 'local_internal_options.conf')
         debug_line = 'agent.debug=2\n'
-    with  open(local_int_conf_path, 'r') as local_file_read:
+    with open(local_int_conf_path, 'r') as local_file_read:
```

Comment:
```suggestion
    with open(local_int_conf_path) as local_file_read:
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb